projects/core/styles/markup/tui-list.less

```diff
@@ -75,6 +75,20 @@
                 background-color: transparent;
             }
         }
+        
+        .tui-list_triangle & {
+            padding-left: @space * 7;
+
+            &:before {
+                content: '◤';
```

Comment:
is it well supported (i am not sure) ?

Or better to use css-entity `\25E4` ?

Reply:
am ok with `\25E4`

Reply:
I added an example, as you requested:
![Screenshot_92](https://user-images.githubusercontent.com/1676900/167106885-89a04c7d-90ff-43de-a50f-3d92b89a111d.png)

I noticed 2 small bugs while working on this ticket:
- Edit on Stackblitz button doesn't work in lists example (check buttons on url https://taiga-ui.dev/lists)
- styles `.tui-list_ordered &` and `.tui-list_linear &` is incorrect, because they change all of their nodes, not only childrens. Correct should be `.tui-list_ordered > &` and `.tui-list_linear > &`

Reply:
Feel free to fix the second issue. Global styles is something that is often gets overlooked since we mostly work on Angular components and related stuff.